emcc.py

```diff
@@ -3728,7 +3728,8 @@ def phase_binaryen(target, options, wasm_target):
     # >=2GB heap support requires pointers in JS to be unsigned. rather than
     # require all pointers to be unsigned by default, which increases code size
     # a little, keep them signed, and just unsign them here if we need that.
-    if settings.CAN_ADDRESS_2GB:
```

Comment:
This line can now be reverted since CAN_ADDRESS_2GB is never set when MEMORY64 is set: See #19755